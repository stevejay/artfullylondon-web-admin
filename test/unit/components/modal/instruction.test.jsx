import React from 'react'
import { shallow } from 'enzyme'

import ModalInstruction from '_admin/components/modal/instruction'

it('should render a narrow container correctly', () => {
  const wrapper = shallow(
    <ModalInstruction><div id='child' /></ModalInstruction>
  )

  expect(wrapper).toMatchSnapshot()
})
