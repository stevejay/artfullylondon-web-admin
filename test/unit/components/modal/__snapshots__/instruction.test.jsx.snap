// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render a narrow container correctly 1`] = `
<p
  className="src-components-modal-instruction__container"
>
  <div
    id="child"
  />
</p>
`;
